Source/Core/Core/BootManager.cpp

```diff
@@ -300,6 +305,9 @@ bool BootCore(const std::string& _rFilename)
         g_wiimote_sources[WIIMOTE_BALANCE_BOARD] = source;
         WiimoteReal::ChangeWiimoteSource(WIIMOTE_BALANCE_BOARD, source);
       }
+
+      IniFile::Section* sysconf_section = game_ini.GetOrCreateSection("Sysconf");
```

Comment:
Having consistent names would of course be nice, but the backwards compatibility issues sound very unfun to deal with. I don't think just expecting users to re-set many of their settings would be acceptable. But either way, it shouldn't be done in this PR.